src/services/relay_renewal_job/refresh_topic_subscriptions.rs

```diff
@@ -46,36 +46,13 @@ pub async fn run(
     let topics_count = topics.len();
     info!("topics_count: {topics_count}");
 
-    // Collect each batch into its own vec, since `batch_subscribe` would convert
-    // them anyway.
-    let topic_batches = topics
-        .chunks(MAX_SUBSCRIPTION_BATCH_SIZE)
-        .map(|chunk| chunk.to_vec())
-        .collect::<Vec<_>>();
-
     // Limit concurrency to avoid overwhelming the relay with requests.
-    const REQUEST_CONCURRENCY: usize = 200;
-
-    futures_util::stream::iter(topic_batches)
-        .map(|topic_batch| {
-            // Map result to an unsized type to avoid allocation when collecting,
-            // as we don't care about subscription IDs.
-            client.batch_subscribe_blocking(topic_batch).map_ok(|_| ())
-        })
-        .buffer_unordered(REQUEST_CONCURRENCY)
-        .try_collect::<Vec<_>>()
-        .await?;
-
-    let elapsed: u64 = start
-        .elapsed()
-        .as_millis()
-        .try_into()
-        .expect("No error getting ms of elapsed time");
-    info!("resubscribe took {elapsed}ms");
+    const REQUEST_CONCURRENCY: usize = 25;
```

Comment:
Is it ok to decrease this almost to 10 times?

Reply:
The variable was re-purposes. It was actually already set to 25